apps/meteor/server/startup/migrations/v305.ts

```diff
@@ -0,0 +1,10 @@
+import { Permissions } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 305,
+	async up() {
+		await Permissions.deleteOne({ _id: 'view-history' });
```

Comment:
The `permissions` collection has `trash`, wonder if we should skip the trash on this delete, like
```
Permissions.col.deleteOne({ _id: 'view-history' })
```

Reply:
Hey @KevLehman , can you elaborate? Not sure I understood what you meant.

Reply:
What i mean is that we have something called `trash` for some DB models. This copies the object before it's deleted into the trash, and then it removes from the original collection.

SO in this case, when removed like this, the permission will be in the trash. Not really sure that's a big concern but would be nice to ask if we can just skip the trash instead for this case.

Reply:
Got it.

@ggazzo , @matheusbsilva137 any thoughts on this?